DC-322-L-1 Web Engineering Lab/utils.py

```diff
@@ -0,0 +1,229 @@
+    print("--------------------------------------\n")
+
+
+def accountExistMsg():
+    """Display account existence error message."""
+
+    print("-------------------------------------------------------------")
+    print("***  Error !! Account Exist Use different 3 digit combo . ***")
+    print("-----------------------------------------------------------\n")
+
+
+def accountNotExistMsg():
+    """Display account non-existence error message."""
+
+    print("------------------------------------------------------------------")
+    print("***  Error !! Account Not Exist Use different 3 digit combo . ***")
+    print("-----------------------------------------------------------------\n")
+
+
+def withdrawAmountMsg():
+    """Display withdrawal confirmation message."""
+
+    print("-----------------------------------------")
+    print("***  You've Withdrawn the ammount  . ***")
+    print("-----------------------------------------")
+
+
+def depositAmmountMsg():
+    """Display withdrawal confirmation message."""
+
+    print("-----------------------------------------")
+    print("***  You've Deposited the ammount  . ***")
+    print("-----------------------------------------")
+
+
+def validateUserInput(min, max, msg):
+    """
+        Validate the user input
+
+        This function will take the min, maz and xustom message
+        and check if the user enter the right value until its
+        right and then retuen the correct value
+
+        Returns:
+        user input
+    """
+
+    while True:
+        try:
+
+            choice = int(input(msg + " : "))
+            if min <= choice <= max:
+                return choice
+            else:
+                print("Invalid Input: Try again")
+        except ValueError:
+            print("Invalid Input: Please enter a valid integer.")
+
+
+def readDataFromFile():
+    """
+        Read Data from the file and store in a variable
+
+        This function will read data from file and store the 
+        data in a list of dictionary and then return it 
+
+        Return:
+        atmData
+    """
+    atmData = []
+
+    with open('atmData.txt', 'r') as file:
+
+        for line in file:
+            parts = line.strip().split(",")
+
+            if len(parts) == 3:
+                accountNumber, accountPassword, balance = parts
+                atmData.append({
+                    "account_number": accountNumber,
+                    "account_password": accountPassword,
+                    "account_balance": balance
+                })
+
+    return atmData
+
+
+def writeDataToFile(accountDetails):
+    """
+        Write data to the file 
+
+        This function will get the list of dictionary 
+        and then convert each dictionary in list and then 
+        in string and overite the existing file it will be 
+        account detail of one new user
+
+        Return:
+        none
+    """
+
+    accountData = [str(item) for item in accountDetails]
```

Comment:
Whenever using such statements, which are not clear to beginner, add some comments. - Always write a code, which could be understood by a child wiith the help of comments.